lib/task_listener.js

```diff
@@ -17,18 +17,29 @@ var EventEmitter = require('events').EventEmitter;
 var util = require('util');
 var TaskQueue = require('./queueservice');
 var exceedsDiskspaceThreshold = require('./util/capacity').exceedsDiskspaceThreshold;
+var VideoDeviceManager = require('./video_device_manager');
+var AudioDeviceManager = require('./audio_device_manager');
+
+const DEVICE_MANAGERS = {
+  'loopbackVideo': new VideoDeviceManager(),
+  'loopbackAudio': new AudioDeviceManager()
```

Comment:
this is nicer then the CPU stuff I did you could probably refactor that out in a similar way